core/os/android/adb/commands.go

```diff
@@ -513,11 +515,21 @@ func (b *binding) PrepareGpuProfiling(ctx context.Context, installedPackage *and
 		log.W(ctx, "Failed to query developer driver: %v, assuming no developer driver found.", err)
 	}
 
+	// Setup debug property.  The GPU driver may enable profiling only if the debug property is set.
```

Comment:
I think it would be better if the comment is: `The GPU driver must enable profiling and have property instrumentation if the debug property is set to 1.`